src/reformatters/common/dynamical_dataset.py

```diff
@@ -41,6 +47,30 @@ def reformat_kubernetes(self) -> None:
         """Run dataset reformatting using Kubernetes index jobs."""
         pass
 
+    def reformat_operational_update(self) -> None:
+        """Update an existing dataset with the latest data."""
+        final_store = self._final_store()
+        tmp_store = self._tmp_store()
+
+        jobs, template_ds = self.region_job_class.operational_update_jobs(
+            final_store=final_store,
+            tmp_store=tmp_store,
+            get_template_fn=self._get_template,
+            append_dim=self.template_config.append_dim,
+            all_data_vars=self.template_config.data_vars,
+            kubernetes_job_name="operational-update",
+        )
+        template_utils.write_metadata(template_ds, tmp_store, get_mode(tmp_store))
+        for job in jobs:
+            process_results = job.process()
+            updated_template = job.update_template_with_results(process_results)
+            template_utils.write_metadata(
+                updated_template, tmp_store, get_mode(tmp_store)
+            )
+            copy_zarr_metadata(updated_template, tmp_store, final_store)
+
+        logger.info(f"Done operational update writing to {final_store}")
```

Comment:
nit: this log line is a bit awkward. 

```suggestion
        logger.info(f"Done writing to {final_store}. Operational update complete.")
```